docs/en/contributor/Contributor-Getting-Started.md

```diff
@@ -294,6 +294,28 @@ Congratulations! You have successfully contributed to Alluxio! Thank you for joi
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/QsbM804rc6Y" frameborder="0" allowfullscreen></iframe>
 
+## Submit Your Feature
+
+If you have a brilliant idea about a new Alluxio feature, we highly encourage you to implement it and 
+contribute to Alluxio repository.
+
+1. Create a Github issue in [Alluxio repository](https://github.com/Alluxio/alluxio/issues)
+regarding your feature.
+1. Attach a feature design document following the [template]({{ '/templates/design-doc-template.docx' | relativize_url }})
```

Comment:
Instead of using .docx format can we simply have a `.txt` template? Using `.docx` makes it more difficult for people to use because it's a proprietary Microsoft format (even if it is pretty ubiquitous). Either that, markdown, or a PDF

Reply:
Changed to a MD file. PTAL ;)

Reply:
So there's pros and cons to using a simple `md` file vs a pdf/docx.

My main concern is also around the use of assets (images) to explain concepts in a design doc. Markdown is good for something that doesn't need images - but for some features, it might be much easier to draw a diagram than to explain it in words.

I would be okay with markdown or PDFs

Reply:
I think MD is good as a template, as it's easy to copy from w/o concern on the rendering. 

I specified the user should use Google doc > MD/PDF file as the design document.